app/Services/Phoenix.php

```diff
@@ -39,4 +39,17 @@ public function postReportback($body = [])
 
         return is_null($response) ? null : $response;
     }
+
+    /**
+     * Send a POST request to save a copy of the signup in Phoenix.
+     *
+     * @param array $body
+     * @return object|false
+     */
+    public function postSignup($body = [], $campaign_id)
+    {
+        $response = $this->post('campaigns/' . $campaign_id . '/signup', $body);
+
+        return is_null($response) ? null : $response;
```

Comment:
Is this ternary necessary? It seems like it'll return `null` only if `$response` is already null.